google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ui/FontUtils.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.ui;
+
+import java.awt.Font;
+import javax.swing.UIManager;
+import javax.swing.text.AttributeSet;
+import javax.swing.text.MutableAttributeSet;
+import javax.swing.text.SimpleAttributeSet;
+import javax.swing.text.StyleConstants;
+import javax.swing.text.StyledDocument;
+
+/** Tools to work with fonts when standard settings are not sufficient. */
+public class FontUtils {
+
+  /**
+   * Converts font for all characters in {@link StyledDocument} to system default font. Does not
+   * change or affect newly added characters.
+   *
+   * @param styledDocument {@link StyledDocument} to update, see {@link javax.swing.JTextPane}
+   */
+  public static void convertStyledDocumentFontToDefault(StyledDocument styledDocument) {
+    for (int ch = 0; ch < styledDocument.getLength(); ch++) {
```

Comment:
super nit: maybe use the standard `i` here instead for readability?

Reply:
seems crazy we have to do this character by character. but I guess we can thank Swing for that

Reply:
Sounds good, I thought `ch` for "character" but it might not be obvious since it's index, not a char.

Reply:
lol, you can do the whole text at once but then we'll loose hyperlink formatting - and looking for hyperlink and default styles and altering them might work but I think they are not mutable as is, so this is the best idea just to change fonts for all styles in the document :)

Reply:
yeah - imo if you wanted to use `ch` or something contextual it should be `charIndex` or something like that (to indicate its an index). but I think `i` is fine too so whatever you prefer.